src/jdk.jconsole/share/classes/sun/tools/jconsole/SummaryTab.java

```diff
@@ -113,6 +113,7 @@ protected void done() {
 
     StringBuilder buf;
 
+    @SuppressWarnings("deprecation")
```

Comment:
Item for future cleanup: refactor the call to `getObjectPendingFinalizationCount()` at line 219 (!) into a local variable declaration, and then move the warnings suppression to that declaration. This reduces the scope of warnings suppression.